src/main.cc

```diff
@@ -159,6 +159,12 @@ static int update_main(LiteClient& client, const bpo::variables_map& variables_m
     version = variables_map["update-name"].as<std::string>();
   }
 
+  // This is only available if -DALLOWED_OPTIONS is set in the CLI args below.
```

Comment:
Comment here should be if -DALLOW_MANUAL_ROLLBACK is set, no?

Wonder if we should also protect this block with an ifdef.

Reply:
Its totally possible. Its essentially dead code, but I did like that its *always* compiled so we'll know of build breaks

Reply:
> Comment here should be if -DALLOW_MANUAL_ROLLBACK is set, no?

corrected.

Reply:
Sure, wfm.